recipes/fontconfig/meson/conanfile.py

```diff
@@ -0,0 +1,137 @@
+from conan import ConanFile
+from conan.tools.files import (
+    apply_conandata_patches,
+    copy,
+    export_conandata_patches,
+    get,
+    rename,
+    replace_in_file,
+    rm,
+    rmdir
+)
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.gnu import PkgConfigDeps
+
+import os
+
+required_conan_version = ">=1.52.0"
+
+
+class FontconfigConan(ConanFile):
+    name = "fontconfig"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Fontconfig is a library for configuring and customizing font access"
+    homepage = "https://gitlab.freedesktop.org/fontconfig/fontconfig"
+    topics = ("fonts", "freedesktop")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
```

Comment:
```suggestion
            self.options.safe_rm("fPIC")
```
once deleted by config_options, it can't be deleted again.